tests/integration/widgets/test_daterange_slider.py

```diff
@@ -57,7 +57,7 @@
 @pytest.mark.selenium
 class Test_DateRangeSlider:
     def test_display(self, bokeh_model_page) -> None:
-        slider = DateRangeSlider(start=start, end=end, value=value, css_classes=["foo"], width=300)
+        slider = DateRangeSlider(start=start, end=end, value=value, value_throttled=value, css_classes=["foo"], width=300)
```

Comment:
Why is it necessary to *set* `value_throttled`? It should really be a readonly property IMO

Reply:
Previously it worked due to implicit conversions (`null == 0`), but making it a readonly property makes more sense than how it works now.

Reply:
I revised readonly properties in 5e8e21c and reverted those changes.